plugins/net_plugin/net_plugin.cpp

```diff
@@ -2485,6 +2483,7 @@ namespace eosio {
          return;
       }
       c->latest_blk_time = std::chrono::system_clock::now();
+      sync_active_time = std::chrono::steady_clock::now(); // reset when we receive a block
```

Comment:
You can do `sync_active_time = c->latest_blk_time = std::chrono::system_clock::now();` to guarantee they are all the same.